python/lsst/obs/subaru/strayLight/rotatorAngle.py

```diff
@@ -248,9 +249,14 @@ def inrStartEnd(header):
         Instrument rotator angle at end of exposure, degrees.
     """
 
-    inst_pa = header.getDouble('INST-PA')
-    ra_t_d = header.getDouble('CRVAL1')
-    de_t_d = header.getDouble('CRVAL2')
+    visitInfo = exposure.getInfo().getVisitInfo()
+    header = exposure.getMetadata()
+
+    inst_pa = 270.0 - visitInfo.getBoresightRotAngle().asAngularUnits(lsst.geom.degrees)
```

Comment:
The 270 has already been applied to getBoresightRotAngle(). See https://github.com/lsst/astro_metadata_translator/blob/master/python/astro_metadata_translator/translators/hsc.py#L132-L138

Reply:
I saw that, but as the straylight code expects the header value of INST-PA, my assumption is that I need to put it back.  I did some tests comparing against the results from before the bug, and the residuals are best with the 270 included.